src/pages/_partials/guide.ejs

```diff
@@ -1,6 +1,13 @@
 <%
 // docs :: [String]
-var docs = public.pages.docs._contents
+// var docs = public.pages.docs._contents
+
```

Comment:
Talk to Eric, going to add a todo, this is going to change but we need something working now.